packages/client/wallets/walletconnect/src/components/common/WC_HACK_Modal.tsx

```diff
@@ -0,0 +1,63 @@
+import { classNames } from "@/utils/classNames";
```

Comment:
Is there a better name for this class...

Reply:
nah, this is very common